pack/fhv/fhvc.json

```diff
@@ -565,5 +658,48 @@
     "text": "<b>(Day [day])</b> You get +1 [intellect]. <b>(Night [night])</b> You get +1 [willpower].\n[reaction] After an investigator at your location draws a [[Blight]] or [[Power]] card from the encounter deck, exhaust The Pearl Diary: That investigator draws 1 card.",
     "traits": "Item. Tome.",
     "type_code": "asset"
+  },
+  {
+    "code": "10726",
+    "encounter_code": "transfiguration",
+    "encounter_position": 1,
+    "faction_code": "mythos",
+    "flavor": "Knotty, scaly growths erupt on the skin, resembling scales or vestigial fingernails.",
+    "illustrator": "Tso",
+    "name": "Strange Mutations",
+    "pack_code": "fhvc",
+    "position": 226,
+    "quantity": 2,
+    "text": "<b>Revelation</b> – Test [willpower] (5). Reduce the difficulty of this test by 1 for each horror on you. If you fail, you must either take 2 horror, or discard 2 cards at random from your hand.",
+    "traits": "Power.",
+    "type_code": "treachery"
+  },
+  {
+    "code": "10727",
+    "encounter_code": "transfiguration",
+    "encounter_position": 3,
+    "faction_code": "mythos",
+    "illustrator": "Stephen Sommers",
+    "name": "Fungal Rot",
+    "pack_code": "fhvc",
+    "position": 227,
+    "quantity": 2,
+    "text": "<b>Revelation</b> – Attach Fungal Rot to an [[Item]] asset you control without Fungal Rot attached. If you cannot, Fungal Rot gains surge. \nTreat attached asset's text box as if it were blank (except for [[Traits]]). \n[action]: Test [combat] (3). If you succeed, discard Fungal Rot. Otherwise, take 1 damage.",
```

Comment:
Extra space before the \n here, and in other spots.

Reply:
Fixed now.